esp/platform/txsummary.cpp

```diff
@@ -90,6 +139,11 @@ void CTxSummary::serialize(StringBuffer& buffer) const
         else
             buffer.appendf("%s;", entry.key.str());
     }
+
+    for (const std::pair<StringBuffer, CumulativeTimer*>& entry : m_timers)
+    {
+        buffer.appendf("%s=%" I64F "ums;", entry.first.str(), entry.second->getTotalMillis());
```

Comment:
the string format is showing up as "%s=%" I64F "ums;" on github, please make sure this is correct.